django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -238,3 +238,47 @@ def typo_checker(df, s, c=0.7):
         return match_dict
     else:
         print('No issues')
+
+def get_validation_error_rows(errors):
+    row_numbers = set()
+    for error in errors:
+        try:
+            row_number = int(error.split()[1][:-1])
+            row_numbers.add(row_number)
+        except (IndexError, ValueError):
+            continue
+    return row_numbers
+
+def validate_phone_numbers(df):
```

Comment:
Given the comment regarding validation functions above, this function should be "standardized" so to speak (as should typo_checker); these functions should have signature `f(df, *columns, **kwargs)`, and their bodies should be updated. We also should standardize what validation functions return; the comment(s) above assume that we return the numbers of rows with errors.